geopaparazzi.app/AndroidManifest.xml

```diff
@@ -195,7 +195,9 @@
         </activity>
         <activity
             android:name=".maps.MapTagsActivity"
-            android:configChanges="orientation|screenSize">
+            android:configChanges="orientation|screenSize"
+            android:launchMode="singleTop"
+            android:parentActivityName=".maps.MapsActivity">
```

Comment:
These additions might not be necessary. When I began, I didn't see this activity referenced and so added it again. When I discovered you already had it, I deleted my addition, but brought over the launchMode and parentActivityName just to be absolutely clear it would not create multiple activity instances (launchMode) or become disconnected from the parent. It probably is fine to remove those two lines.